packages/app/components/card/card.tsx

```diff
@@ -89,12 +92,15 @@ function Card({
         // TODO: add accessibility types for RNW
         accessibilityRole="article"
         dataSet={Platform.select({ web: { testId: "nft-card" } })}
-        style={{
-          // @ts-ignore
-          boxShadow: isDark ? CARD_DARK_SHADOW : CARD_LIGHT_SHADOW,
-        }}
+        style={[
+          {
+            // @ts-ignore
+            boxShadow: isDark ? CARD_DARK_SHADOW : CARD_LIGHT_SHADOW,
+          },
+          sizeStyle,
+        ]}
         tw={[
-          size,
+          !sizeStyle ? size : "",
```

Comment:
not very happy about this solution. I think later we should convert media/card to explicitly pass height/width from outside.